course-recommendation.md

```diff
@@ -0,0 +1,165 @@
+<script>
+function googleTranslateElementInit() {
+  new google.translate.TranslateElement({pageLanguage: 'zh'}, 'google_translate_element');
+}
+</script>
+<script async src="https://translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script>
+
+<div style="padding-bottom: 6px">
+<a href="http://www.mingcns.org">返回文档索引主页</a>
+</div>
+
+# 新文档模板
+
+{:.no_toc}
+
+<div align="right">
+本指南非官方文件，详见<a href="#版权与免责协议">版权与免责协议</a><br>
+(Version: 2000-01-01)
+</div><br>
+
+## 目录
+
+{:.no_toc}
+
+- 目录
+  {:toc}
+
+## 通识水课推荐：
+
+<table>
+  <tr>
+    <th>课号</th>
+    <th>课程名称</th>
+    <th>满足标签</th>
+  </tr>
+  <tr>
+    <td>JOUR 1501 *Online</td>
+    <td>Digital Games and Society</td>
+    <td>Arts/Humanities & Technology and Society</td>
+  </tr>
+  <tr>
+    <td>MUS 1013 *Oneline</td>
+    <td>Rock I: The Historical Origins and Development of Rock Music to 1970</td>
+    <td>Arts/Humanities Diversity & Soc. Justice in the U.S</td>
+  </tr>
+  <tr>
+    <td>AMST 2011</td>
+    <td>The United States since September 11</td>
+    <td>Civic Life & Ethics Historical Perspectives</td>
+  </tr>
+  <tr>
+    <td>AMST 2031</td>
+    <td>Chasing the American Dream: Economic Opportunity & Inequality in the U.S.</td>
+    <td>Diversity and Soc. Justice in the U.S. & Historical Perspectives</td>
+  </tr>
+  <tr>
+    <td>HSCI 1011</td>
+    <td>Digital World</td>
+    <td>Historical Perspectives & Technology and Society</td>
+  </tr>
+  <tr>
+    <td>CI 1124</td>
+    <td>Global Stories of Education: Literature for Young Adults</td>
+    <td>Global Perspectives & Literature</td>
+  </tr>
+  <tr>
+    <td>ECON 1101</td>
+    <td>Principles of Microeconomics</td>
+    <td>Global Perspectives & Social Sciences</td>
+  </tr>
+  <tr>
+    <td>BBE 1002</td>
+    <td>Biorenewable Resources</td>
+    <td>Technology and Society</td>
+  </tr>
+  <!-- <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr> -->
+  <!-- <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr> -->
+</table>
+
+<!-- TODO -->
+<!-- _持续更新中...有推荐可直接联系---(Decide a person to contact or form to submit_ -->
+
+## 版权与免责协议
+
+本项目包含的所有内容均为通用情况下的知识普及目的而创作， **并且不构成任何法律上的建议** 。
+关于税务、移民身份相关问题的法律上的建议，请咨询专业会计师、律师或者 UMN ISSS 官方。
+由于信息变化较快以及作者能力所限，本文内容无法保证 100%的准确性，敬请谅解。
+欢迎通过 Github Pull Request 来对本文档的更改做出建议。
+
+This document and any content in “Project MinG/CNS” are general introductions for readers only,
+and shall not constitute nor be relied on as legal opinion or legal advice in any form.
+We assume no liability for anything herein.
+If you need any legal or tax help, please talk to a tax, legal or ISSS immigration advisor immediately.
+
+Suggestions for updating and/or fixing are always welcomed via Github Pull Request.
+
+本文全文著作权属于联合志愿作者团队以及共同创作者。
+
+The “copyleft” of the entire document belongs to all of the author team, co-editors, and contributors.
+
+参与本文编撰的作者与共同创作者名单(排名不分先后):  
+Authors, co-editors, and contributors of this documentation:
+
+- Zheng Robert Jia
```

Comment:
We use "Last Name, Names" format in this repo, but if you prefer "Names Last Name" format, this is also OK